test/tools/ossfuzz/config/strict_assembly.dict

```diff
@@ -0,0 +1,118 @@
+" -> "
+" := "
+"("
+")"
+", "
+":"
+" dup1 "
```

Comment:
explicit dupN opcodes are not allowed in strict assembly.

Reply:
Considering `jump(r)` resulted in a failed assertion, shouldn't we leave these (including `dup`) for now with the hope that we fix other potential errors due to wrong expectations encoded in the compiler?

Reply:
We mostly want to find problems with the optimizer and because of that the grammar should focus on generating input that passes the parser, shouldn't it?

Reply:
I agree. But this `{jump(r)}` causes an internal compiler exception inside the parser. It should be a graceful parser error (return false instead of throwing an assertion), shouldn't it?